department-of-veterans-affairs/caseflow
# Shalliburton/caseflow 396 add hearing schedule info

Resolves [CASEFLOW-396](https://vajira.max.gov/browse/CASEFLOW-396)

### Description
Please explain the changes you made here.

### Acceptance Criteria
- [x] Change the Scheduled Hearings tab from a table to a list of scheduled hearing cards
- Scheduled hearing cards should display the following information
  - On left
    - [x] Hearing time in RO time zone
    - [x] Hearing time in Eastern time zone (only if RO time zone is not Eastern)
  - First line on right
    -  [x] Veteran/appellant name
    -  [x] Veteran ID: <#>
  - Second line on right
    -  [x] Number of issues
    -  [x] Docket number
    -  [x] POA name
  - Third line on right
    -  [x] Appeal stream type
    -  [x] Hearing type (Virtual, Video, Central)
    -  [x] "View Case Details" link that links to Case Details page
-  [x] Cards should be displayed in order of hearing time


### Testing Plan
1. Shadow user `BVASYELLOW`: http://localhost:3000/test/users
2. Navigate to the hearings schedule: http://localhost:3000/hearings/schedule
3. Click the "Schedule Veterans" button
4. Ensure that the list of items matches the Figma diagram

### User Facing Changes

 BEFORE|AFTER
 ---|---
|<img width="1245" alt="Screen Shot 2021-04-08 at 11 07 44 AM" src="https://user-images.githubusercontent.com/61989526/114075698-ae14a800-985a-11eb-8aa9-78ca8e52d4e2.png">|<img width="1238" alt="Screen Shot 2021-04-08 at 11 06 02 AM" src="https://user-images.githubusercontent.com/61989526/114075554-81609080-985a-11eb-8dd7-a451e0e1fa7e.png">
|<img width="1251" alt="Screen Shot 2021-04-08 at 11 07 51 AM" src="https://user-images.githubusercontent.com/61989526/114075719-b4a31f80-985a-11eb-8d71-31833b4fcd54.png">|<img width="1247" alt="Screen Shot 2021-04-08 at 11 06 12 AM" src="https://user-images.githubusercontent.com/61989526/114075562-845b8100-985a-11eb-8d47-8e3bd2a5a34e.png">


### Code Documentation Updates

### Storybook Story
*For Frontend (Presentationa) Components*
